### PR TITLE
Add a block to switch between Anna and Elsa in frozen artist

### PR DESCRIPTION
**Pull request by balderdash**
image

Switching characters also changes the line pattern, so you can mix it up:

image

The implementation basically just swaps out the loaded skin, re-initializing the images that change (except the background, which keeps whatever the initial skin used).

### CODE REVIEW

File: apps/i18n/turtle/en_us.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
   "penUp": "pencil up",
   "reinfFeedbackMsg": "Here is your drawing! Keep working on it or continue to the next puzzle.",
   "setAlpha": "set alpha",
+  "setCharacter": "Set Character to {character}",
```
Comment (joshlory) [inline apps/i18n/turtle/en_us.json]: Tiny nit: lowercase to match other blocks?

File: apps/src/turtle/turtle.js [hunk 2]
```diff
@@ -275,23 +277,38 @@ Artist.prototype.init = function (config) {
...
+    removedBlock = true;
+    next = setArtistBlock.querySelector('next');
+    let parentNext = setArtistBlock.parentNode;
+    let parentBlock = parentNext.parentNode;
+    parentBlock.removeChild(parentNext);
+    if (next) {
+      parentBlock.appendChild(next);
+    }
+    setArtistBlock = blocksDom.querySelector('block[type="turtle_setArtist"]');
+  }
```
Comment (joshlory) [inline apps/src/turtle/turtle.js:356]: Why remove set artist blocks while remixing?

File: apps/src/turtle/turtle.js [hunk 3]
```diff
@@ -1175,6 +1180,14 @@ Artist.prototype.step = function (command, values, options) {
       this.ctxScratch.restore();
 
       break;
+    case 'setArtist':
+      if (this.skin.id !== values[0]) {
+        this.skin = ArtistSkins.load(this.studioApp_.assetUrl, values[0]);
+        this.loadTurtle();
```
Comment (joshlory) [inline apps/src/turtle/turtle.js]: It looks like [this calls `Artist.prototype.display` on `this.avatarImage.onload`](https://github.com/code-dot-org/code-dot-org/blob/be77b537e99650366a6dbd50c6bf531345508702/apps/src/turtle/turtle.js#L537).  Will we end up drawing pre-draw, etc. twice?

**Comment by joshlory** [comment]
+cc @ryansloan for visibility.

File: apps/src/turtle/turtle.js (hunk 2 above)
Comment (balderdash) [reply apps/src/turtle/turtle.js:356]: When you remix a frozen share, it gets converted to a standard artist project. I assumed we only wanted Elsa and Anna to be available on frozen levels.

If that's not the case, there's probably more work to be done to get Anna and Elsa working on levels that didn't start out as frozen, but I haven't tried it yet.

File: apps/src/turtle/turtle.js (hunk 3 above)
Comment (balderdash) [reply apps/src/turtle/turtle.js]: Yep, fixed.

File: apps/i18n/turtle/en_us.json (hunk 1 above)
Comment (balderdash) [reply apps/i18n/turtle/en_us.json]: Done.

**Comment by ryansloan** [comment]
LGTM

**Pull request merged**